File: Back-end/Squirrel.Data/Entities/Transaction.cs
namespace Squirrel.Data.Entities
{
    public class Transaction
    {
        public int Id { get; set; }
        public DateTime Time { get; set; }
        public double Amount { get; set; }
        public string Description { get; set; }

        public int CategoryId { get; set; }
        public Category Category { get; set; }

    }
}

File: Mobile/app/src/main/java/com/example/squirrel/entities/Transaction.kt
package com.example.squirrel.entities

import kotlinx.datetime.DateTimeUnit
import kotlinx.serialization.Serializable
@Serializable
data class Transaction(
    var id: Int,
    var time: kotlinx.datetime.LocalDateTime,
    var amount: Double,
    var description: String,
    var categoryId: Int
)

